documentation/source/users/cantherm/input_pdep.rst

```diff
@@ -285,20 +631,25 @@ be treated as a product channel. An example of the ``network`` block is shown be
         }
     )
 
+.. image:: acetyl+O2.jpg
+
+Image source: `J.W. Allen, PhD dissertation, MIT 2013 <http://hdl.handle.net/1721.1/81677>`_,
+calculated at the RQCISD(T)/CBS//B3LYP/6-311++G(d,p) level of theory
+
 Algorithm Parameters
 ====================
 
 The overall parameters for the pressure-dependence calculation must be defined in a
-``pressureDependence`` block at the end of the input file.   The following parameters are necessary:
+``pressureDependence`` function at the end of the input file.   The following parameters are necessary:
 
 
 
 ====================== ====================================================================================================================================================
 Parameter              Description
 ====================== ====================================================================================================================================================
 ``label``              Use the name for the ``network`` declared previously
-``method``             Method to use for calculating the pdep network. Use either 'modified strong collision', 'reservoir state', or 'chemically-significant eigenvalues'
-``interpolationModel`` Select the output type for the pdep kinetics, either in 'chebyshev' or 'pdeparrhenius' (plog) format
+``method``             Method to use for calculating the pdep network. Use either ``'modified strong collision'``, ``'reservoir state'``, or ``'chemically-significant eigenvalues'``
+``interpolationModel`` Select the output type for the pdep kinetics, either in ``'chebyshev'`` or ``'pdeparrhenius'`` (plog) format
 ``activeKRotor``       A flag indicating whether to treat the K-rotor as active or adiabatic
 ``activeJRotor``       A flag indicating whether to treat the J-rotor as active or adiabatic
 ====================== ====================================================================================================================================================
```

Comment:
Ok, to be consistent we probably should add the missing parameters to the table (Tmin,  Tmax, etc.), except for ``rmgmode`` . The Tlist and Plist options are currently not mentioned at all, so that should also be added. I'll take care of it.